modules/programs/oh-my-posh.nix

```
@@ -0,0 +1,81 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+
+  cfg = config.programs.oh-my-posh;
+
+  configArgument =
+    optionalString (cfg.configFile != null) "--config ${cfg.configFile}";
+
+  themeArgument =
+    optionalString ((cfg.configFile == null) && (cfg.useTheme != null))
+    "--config ${cfg.package}/share/oh-my-posh/themes/${cfg.useTheme}.omp.json";
```

Perhaps something more straight forward would be simpler:

```suggestion
  configArgument =
    if cfg.configFile != null then
      "--config cfg.configFile"
    else if cfg.useTheme != null then
      "--config ${cfg.package}/share/oh-my-posh/themes/${cfg.useTheme}.omp.json"
    else
      "";
```

---

That is definitely nicer. Thank you!